server/db/helper.js

```diff
@@ -67,9 +67,18 @@ const getCourse = (id) => {
     });
 };
 
+const getUserBadges = (userId) => UserBadge.findAll({ 
+  where: {
+    id: userId,
```

Comment:
You want to find the UserBadge connection where the idUser matches what you pass in to the function